llm-disease-categorization/conf/base/parameters.yml

```diff
@@ -0,0 +1,73 @@
+categorization_prompt: Identify and return ONLY (provide no additional text) the best-fitting category of the following list of categories for the disease
+
+disease_categories_txgnn_modified:
+  name: txgnn
+  categories:
+    - adrenal_gland_disease
+    - neurodegenerative_disease
+    - metabolic_disorder
+    - cardiovascular_disorder
+    - autoimmune_diseases
+    - mental_health_disorder
+    - anemia
+    - cancer
+    - inflammatory_disease
+    - allergy
+    - other
+
+disease_categories_anatomical:
+  name: anatomical
+  categories:
+    - biliary_disorder
+    - blood_bone_marrow_disorder
+    - bone_disorder
+    - brain_disorder
+    - ear_disorder
+    - endocrine_disorder
+    - eye_disorder
+    - heart_disorder
+    - immune_disorder
+    - joint_disorder
+    - kidney_disorder
+    - liver_disorder
+    - lower_gastrointestinal_disorder
+    - lung_disorder
+    - lymphatic_disorder
+    - muscle_disorder
+    - nose_disorder
+    - reproductive_system_disorder
+    - skin_disorder
+    - spinal_disorder
+    - stomach_disorder
+    - teeth_disorder
+    - throat_disorder
+    - upper_gastrointestinal_disorder
+    - urinary_tract_disorder
+    - vascular_disorder
+    - other
+
+disease_categories_medical_specialization:
+  name: medical_specialization
```

Comment:
Where did you get these from? Is there an official source?

Reply:
A few rounds of iteration with @elliottsharp